### PR TITLE
Add some stream on id.m3u

### PR DESCRIPTION
**Pull request by singkie**

### CODE REVIEW

File: streams/id.m3u [hunk 1]
```diff
@@ -99,6 +107,8 @@ https://h1.intechmedia.net/intech/ch29.m3u8
 #EXTINF:-1 tvg-id="JayaTVJayapura.id" user-agent="Mozilla/5.0 (X11; Linux x86_64)",Jaya TV (360p) [Geo-blocked]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (X11; Linux x86_64)
 https://h1.intechmedia.net/intech/ch52.m3u8
+#EXTINF:-1 tvg-id="JekTV.id",Jek TV (720p)
+http://61.8.64.138:1935/jektv/myStream/playlist.m3u8
```
Comment (freearhey) [inline streams/id.m3u:111]: Offline.

image

File: streams/id.m3u [hunk 2]
```diff
@@ -10,11 +10,17 @@ https://alfa.kugo.id/playout/ainos.stream/chunklist.m3u8
...
+https://tv.yankie.my.id/cdn/antv.m3u8
 #EXTINF:-1 tvg-id="AshiilTV.id" user-agent="Mozilla/5.0 (X11; Linux x86_64)",Ashiil TV (360p) [Geo-blocked]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (X11; Linux x86_64)
 https://h1.intechmedia.net/intech/ch2.m3u8
 #EXTINF:-1 tvg-id="BaliTV.id",Bali TV (720p) [Geo-blocked]
 https://cdn-telkomsel-01.akamaized.net/Content/HLS/Live/channel(8e867ae0-b2c0-4968-9f60-a11aee8c0987)/index.m3u8
+#EXTINF:-1 tvg-id="BandungTV.id",Bandung TV
+http://202.150.153.254:65500/bandungtvWEBSITE.m3u8
+#EXTINF:-1 tvg-id="BantenTV.id",Banten TV (720p)
+https://5bf7b725107e5.streamlock.net/bantentv/bantentv/playlist.m3u8
```
Comment (freearhey) [inline streams/id.m3u:23]: Offline.

image

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

File: streams/id.m3u (hunk 2 above)
Comment (BellezaEmporium) [reply streams/id.m3u:23]: https://bantentv.com/live/ seems to be from an official source, maybe not 24/7.

File: streams/id.m3u (hunk 1 above)
Comment (BellezaEmporium) [reply streams/id.m3u:111]: Also seems to be from an official source : https://jektv.co.id/live/

**Comment by singkie** [comment]
> Please check the marked links again.

i've fixed this. it's not 24/7, and i've already add this information

File: streams/id.m3u (hunk 2 above)
Comment (singkie) [reply streams/id.m3u:23]: @BellezaEmporium @freearhey yeah, it's not 24/7

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**